src/components/Footer.js

```diff
@@ -1,31 +1,35 @@
 import React from 'react';
 import links from '../constants/links';
 
+const d = new Date();
+const year = d.getFullYear();
+
 const Footer = (props) => {
   return (
-    <footer id="footer">
-      <div className ="block-center">
-        <div className="social-icon-group">
-          <table>
-            <tr>
-              <td>
-                <a href={links.github} className="social-icon">
-                  <i className="fa fa-github fa-2x"></i>
-                </a>
-              </td>
-              <td>
-                <a href={links.email} className="social-icon">
-                  <i className="fa fa-envelope fa-2x"></i>
-                </a>
-              </td>
-            </tr>
-          </table>
+    <footer id="footer" className="footer">
+      <div className="footerText">
+        <div className="footerCopyright">&copy; {year} Adopt A Family</div>
```

Comment:
I do not think making the copyright dynamic to current year is the right way to go